project/BuildVars.scala

```diff
@@ -2,11 +2,11 @@ import sbt._
 
 object BuildVars {
   lazy val awsVersion         = "1.11.48"
-  lazy val contentAtomVersion = "2.4.14"
+  lazy val contentAtomVersion = "2.4.16"
```

Comment:
eeeeep! Would me merging in https://github.com/guardian/media-atom-maker/pull/108/files#diff-a7d6ef61e0ad0403cd2d8180afdcbe42R5 cause pain?